components/suggestion/provider.tsx

```diff
@@ -10,21 +27,22 @@ export default class Provider {
         this.forceDispatch = false;
     }
 
-    handlePretextChanged(pretext, callback) { // eslint-disable-line no-unused-vars
+    // eslint-disable-next-line
```

Comment:
Could you please only disable the rule that is being violated and not all of eslint?